app/components/dsfr_component/breadcrumbs_component/breadcrumb_component.rb

```diff
@@ -0,0 +1,24 @@
+class DsfrComponent::BreadcrumbsComponent::BreadcrumbComponent < DsfrComponent::Base
+  def initialize(label:, link: nil, classes: [], html_attributes: {})
```

Comment:
nitpick but maybe `href` would be more explicit than `link`. alternatives : `path`, `url`, `path_or_url`

Reply:
yeah `href` is better